bindings/go/go.mod

```diff
@@ -0,0 +1,7 @@
+module opendal.apache.org/go
+
+go 1.20
+
+require github.com/ebitengine/purego v0.4.0-alpha.4
```

Comment:
This should be https://github.com/ebitengine/purego/releases/tag/v0.3.2

Instead of using 0.4.0 which is an alpha release

Reply:
This pull request will not be merged in its current form. Please prioritize other more important issues. Thank you.